datasets/tedlium2.py

```diff
@@ -214,3 +214,63 @@ def load_stm_data(self, stm_file):
                 text = " ".join(tokens[6:])
                 data.append([rec_name, channel, spk_name, start, end, gender, text])
         return data
+
+
+class CreateTEDLIUM2BlissCorpusJobV2(CreateTEDLIUM2BlissCorpusJob):
+    """
+    It has the same logic as CreateTEDLIUM2BlissCorpusJob but the corpus name is distinct for each partition
+    """
+
+    def __init__(self, corpus_folders):
+        super().__init__(corpus_folders)
```

Comment:
Not needed if nothing is changed. By default the parent `__init__` will be used anyway.

```suggestion
```